database/seeds/DatabaseSeeder.php

```diff
@@ -11,6 +11,6 @@ class DatabaseSeeder extends Seeder
      */
     public function run()
     {
-        $this->call(RolesAndPermissionsSeeder::class);
+        //
```

Comment:
Can we remove empty method?

Reply:
That's there out of the box. I'd rather leave it in case we do want to have more seeds in the future. No harm there by leaving it.